lib/types/device-type-json.ts

```diff
@@ -1,12 +1,9 @@
+import type { Contract } from './contract';
 import type { Dictionary } from '../../typings/utils';
 
 /* types for the /device-types/v1 endppoints */
 
-export interface DeviceType {
-	slug: string;
-	name: string;
-	aliases: string[];
-
+export interface DeviceType extends Contract {
```

Comment:
Since this is exactly matching the device-type.json types, and also requires us to use `?.` whenever accessing `name` from now on, let's completely revert the changes in this file.
I hope we would actually be able to completely drop this file at some point...
If for the contracts work you can declare a more accurate DeviceType Contract definition, then let's do it in the new `lib/types/contract.ts` file w/ a specialized `DeviceTypeContract` interface.